.mergify.yml

```diff
@@ -34,9 +22,9 @@ pull_request_rules:
   - name: Add deleted-assets label if status checks fail
     conditions:
       - or:
-        - check-failure="Check if assets were deleted on PR opened"
-        - check-failure="Check if assets were deleted on PR sync"
+          - check-failure="Check if assets were deleted on PR opened"
+          - check-failure="Check if assets were deleted on PR sync"
```

Comment:
Are these extra indentation intentional?

Reply:
No, I think my editor added them, but I can remove it so there's less diff

Reply:
yep, let's do it!